src/types.ts

```diff
@@ -8,6 +8,7 @@ export interface MirrorOptions {
   mirror?: string;
   customDir?: string;
   customFilename?: string;
+  baseOnly?: boolean;
```

Comment:
Can you add some docs for this property? I know that the others don't have docs, but that's an oversight. 😢

Reply:
@malept i think that should do it, let me know if you'd like more!